tests/golden/check-localhost/check-localhost.bats

```diff
@@ -8,30 +8,15 @@ load '../helpers/bats-support/load'
 load '../helpers/bats-assert/load'
 load '../helpers'
 
-@test "Config: check-localhost=false, CLA --check-localhost not provided" {
+@test "Ignore localhost" {
```

Comment:
I guess `check-ignoreRefs` would now be a more appropriate name for this folder + bats file :thinking: